packages/hyperion-autologging-visualizer/src/component/ALSessionGraph.react.tsx

```diff
@@ -35,7 +35,7 @@ type EventBody = {
   channelEventName?: string,
   callFlowlet?: IALFlowlet | null,
   copiedFlowlet?: CopiedFlowlet,
-  targetElement?: HTMLElement | null,
+  targetElement?: HTMLElement | SVGElement | null,
```

Comment:
We should simply convert the type to `Element` which is the parent type of HTML and SVG element.